### PR TITLE
docs(admin-api) add certificates and snis reference

### PR DESCRIPTION
**Pull request by p0pr0ck5**

### CODE REVIEW

File: app/docs/0.10.x/admin-api.md [hunk 1]
```diff
@@ -1081,6 +1095,346 @@ HTTP 200 OK
...
+```
+
+---
+
+## SNI Objects
+
+An SNI object represents a many-to-one mapping of hostnames to a certificate.
+That is, a certificate object can have many hostnames associated with it; when
+Kong receives an SSL request, it uses the SNI field in the Client Hello to
+lookup the certificate object based on the `sni` associated with the certificate.
```
Comment (thibaultcha) [inline app/docs/0.10.x/admin-api.md]: "SNI"? We never refer to entities as code blocks.

File: app/docs/0.10.x/admin-api.md [hunk 2]
```diff
@@ -1081,6 +1095,346 @@ HTTP 200 OK
...
+
+---
+
+### Update SNI
+
+<div class="endpoint patch">/snis/{name}</div>
+
+Attributes | Description
+---:| ---
+`name`<br>**required** | The name of the SNI object
```
Comment (thibaultcha) [inline app/docs/0.10.x/admin-api.md]: dot

File: app/docs/0.10.x/admin-api.md [hunk 3]
```diff
@@ -1081,6 +1095,346 @@ HTTP 200 OK
...
+
+---
+
+### Delete SNI
+
+<div class="endpoint delete">/snis/{name}</div>
+
+Attributes | Description
+---:| ---
+`name`<br>**required** | The name of the SNI object
```
Comment (thibaultcha) [inline app/docs/0.10.x/admin-api.md]: dot (again, consistency)

**Comment by robert-zenedge** [comment]
Thanks @thibaultcha! Reference page and docs nav updated.

**Comment by p0pr0ck5** [comment]
Bah, using an old laptop, ignore the previous comment from the wrong github account :p Reference docs and nav updated accordingly!

File: app/_data/docs_nav_0.10.x.yml [hunk 4]
```diff
@@ -170,6 +170,48 @@
         - text: Retrieve Plugin Schema
           url: /admin-api/#retrieve-plugin-schema
 
+    - text: Certificate Object routes
+      url: /admin-api/#certificate-object
+      items:
+        - text: Add Certificate
+          url: /admin-api/#add-certificate
+
+        - text: Retrive Certificate
```
Comment (thibaultcha) [inline app/_data/docs_nav_0.10.x.yml]: typo: "retrive" :)

File: app/docs/0.10.x/admin-api.md [hunk 5]
```diff
@@ -43,6 +43,20 @@ upstream_body: |
     `name` | This is a hostname like name that can be referenced in an `upstream_url` field of an `api`.
     `slots`<br>*optional* | The number of slots in the loadbalancer algorithm (`10`-`65536`, defaults to `1000`).
     `orderlist`<br>*optional* | A list of sequential, but randomly ordered, integer numbers that determine the distribution of the slots in the balancer. If omitted it will be generated. If given, it must have exactly `slots` number of entries.
+
+certificate_body: |
+    Attributes | Description
+    ---:| ---
+    `cert` | PEM-encoded public certificate of the SSL key pair.
+    `key` | PEM-encoded private key of the SSL key pair.
+    `snis`<br>*optional* | One or more hostnames to associate with this certificate as an SNI. This is a sugar parameter that will, under the hood, create an SNI Object and associate it with this certificate for your convenience.
```
Comment (thibaultcha) [inline app/docs/0.10.x/admin-api.md]: I think we never capitalize "object" in "SNI obeject"

**Comment by thibaultcha** [comment]
Merged and deployed, thank you!

**Pull request closed without merging**